config/cmake/h4config.h.in

```diff
@@ -59,35 +50,26 @@
 /* Define to 1 if you have the `z' library (-lz). */
 #cmakedefine H4_HAVE_LIBZ @H4_HAVE_LIBZ@
 
-/* Define to 1 if you have the <memory.h> header file. */
-#cmakedefine H4_HAVE_MEMORY_H @H4_HAVE_MEMORY_H@
-
 /* Define if we export HDF4-built unmangled netCDF 2.3.2 API calls */
 #cmakedefine H4_HAVE_NETCDF @H4_HAVE_NETCDF@
 
-/* Define to 1 if you have the <netdb.h> header file. */
-#cmakedefine H4_HAVE_NETDB_H @H4_HAVE_NETDB_H@
-
 /* Define to 1 if you have the <netinet/in.h> header file. */
 #cmakedefine H4_HAVE_NETINET_IN_H @H4_HAVE_NETINET_IN_H@
 
-/* Define to 1 if you have the <resolv.h> header file. */
-#cmakedefine H4_HAVE_RESOLV_H @H4_HAVE_RESOLV_H@
-
-/* Define to 1 if you have the <stddef.h> header file. */
-#cmakedefine H4_HAVE_STDDEF_H @H4_HAVE_STDDEF_H@
-
 /* Define to 1 if you have the <stdint.h> header file. */
-#cmakedefine H4_HAVE_STDINT_H @H4_HAVE_STDINT_H@
+#define H4_HAVE_STDINT_H 1
+
+/* Define to 1 if you have the <stdio.h> header file. */
+#define H4_HAVE_STDIO_H 1
```

Comment:
The Autotools define some standard library header symbols regardless of what we do in configure.ac, so I'm duplicating that here, but without bothering to check if C89/99 are present

Reply:
Maybe change the comment that the setting is constant.  So someone in the future doesn't try to undo it?